recipe/meta.yaml

```diff
@@ -20,16 +19,17 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - patch     # [(not win) and (py==311)]
-    - m2-patch  # [win and (py==311)]
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - python
-    - pip
+    - pip <23.1
+    - setuptools
+    - wheel
     - libprotobuf {{ version }}
   run:
     - python
-    - libprotobuf {{ version }}
```

Comment:
I would add it back like this `- libprotobuf  # pin handled through libprotobuf's run_exports` so that it's more explicit.

Reply:
done